.github/workflows/lint.yml

```
@@ -0,0 +1,31 @@
+name: lint
+
+on:
```

yamllint

[warning] truthy value should be one of [false, true]

(truthy)